System-Architecture/CSC-Overview/Main-Telescope-CSCs/index.rst

```diff
@@ -1,33 +1,103 @@
-        :name: fig-label
-        :target: http://target.link/url
-        Caption text.
+.. warning::
+    This page is under heavy development and is subject to change.
+
+As mentioned in the :ref:`Control-Packages-General-CSC-Overview`, it is convenient to group CSCs into one of three categories, one of which is the Main Telescope (or MT).
+All CSCs associated with the Main Telescope receive an ``MT`` prefix to their name, such as the MTMount or MTDome.
+CSCs in this group are strictly associated with the operation and calibration of the Main Telescope and cannot be moved between systems.
+
+Although a full list of Main Telescope CSCs, indicated by the ``MT`` prefix are found in the `Master CSC Table <https://ts-xml.lsst.io/#master-csc-table>`__, it is generally most useful to associate the CSCs by the control-package it is associated with, as it done below.
+
+Links are provided to the user guide when available. Interface information for each CSC can be found in the `ts-xml guide <https://ts-xml.lsst.io/>`__.
+Due to the large number of CSCs, they are arranged according to their :ref:`high-level control class <System-Architecture-Control-Packages>`, when one exists.
+
+Main Telescope Control System (MTCS)
+====================================
+
+.. _Dome: https://ts-dome.lsst.io/
+.. _MTDomeTrajectory: https://ts-mtdometrajectory.lsst.io/
+.. _MTMount: https://ts-mtmount.lsst.io/
+.. _MTAOS: https://ts-mtaos.lsst.io/
+
+The :ref:`Main <Control-Packages-MainTel>` is a high-level observatory control package that provides the most user-friendly interface to the Main Telescope related CSCs related to operating the telescope and dome.
+
+The following MT CSCs are associated with that class, but also controllable individually if required:
+
+MTCamera
+^^^^^^^^
+
+    * The MTCamera CSC is used to take images with the LSST Camera.
+
+`Dome`_
+^^^^^^^^^
+
+    * The `Dome`_ CSC is responsible for the positioning of the main telescope dome and the light and windscreen.
+      It also handles the positioning the louvers and opening and closing the dome shutter.
+      The logic that determines the ideal positioning of the dome and trajectory is handled externally in the MTDomeTrajectory CSC.
+
+`MTDomeTrajectory`_
+^^^^^^^^^^^^^^^^^^^
+
+    * The `MTDomeTrajectory`_ CSC controls how and when the dome moves relative to the telescope position. It essentially monitors the telescope position and trajectory and sends commands to the MTDome CSC when appropriate to do so.
+
+`MTMount`_
+^^^^^^^^^^
+
+    * The MT mount control system performs the servo control of the telescope mount motors and encoders.
+        This includes the camera cable wrap.
+
+MTPtg
+^^^^^
+
+    * The MT pointing component CSC converts the celestial position to mount coordinates (altitude and azimuth) and    includes an analytical model of the mount to increase pointing and tracking accuracy.
+      It sends position and velocity commands to the `MTMount`_.
+
+Rotator
```

Comment:
Another CSC that will need renaming.. I see you mention `MTRotator` in the text. I suggest it says `Rotator` for now and note that it is going to change